content/javascript/concepts/dates/terms/getUTCDate/getUTCDate.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '.getUTCDate()' 
+Description: 'Returns the day of the month for the provided date' 
+Subjects: 
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`getUTCDate()`** returns the day of the month for the provided date according to universal time.
+
+## Syntax
+
+```js
+myDate.getUTCDate()
+```
+
+The `getUTCDate()` method returns the day of the month for the provided date `myDate`.
+
+## Example
+
+In the example below, the variable `myDate` stores the day of the month returned from a new Date object and logs to the console.
+
+```js
```

Comment:
It may be preferable to construct a specific date in the example (so that the result is fixed) and include a shell block with the output

Reply:
I have made the updates as suggested @caupolicandiaz but not sure what shell block means so don't know how to do that.